docs/en_US/SQuAD_evolution_examples.md

```diff
@@ -1,31 +1,17 @@
 # Automatic Model Architecture Search for Reading Comprehension
```

Comment:
This comment is not to this single md, but to the entire PR.

may we add a ReadMe.md to the nni/examples folder, so that users who directly go to that root can figure out the examples in a glance